Sources/Units/Unit/Fraction.swift

```diff
@@ -0,0 +1,163 @@
+
+
+public struct Fraction: Hashable, Equatable, Sendable {
+    let numerator: Int
+    let denominator: Int
+
+    init(numerator: Int, denominator: Int) {
+        let gcd = Self.gcd(numerator, denominator)
+        self.numerator = numerator / gcd
+        self.denominator = denominator / gcd
```

Comment:
Should this be public?

Reply:
Yes. I guess the tests do a testable import so I didn't catch that.

Reply:
I've made some more things public